var/spack/repos/builtin/packages/wannier90/package.py

```diff
@@ -16,7 +17,11 @@ class Wannier90(MakefilePackage):
     """
     homepage = 'http://wannier.org'
     url = 'http://wannier.org/code/wannier90-2.0.1.tar.gz'
+    # Newer versions are here
+    new_url = 'https://github.com/wannier-developers/wannier90/archive/v3.1.0.tar.gz'
```

Comment:
This url doesn't appear to be used anywhere. I would replace the `url` above with this one so that `spack versions` and `spack checksum` will work correctly.